source/user/eph.rst

```diff
@@ -0,0 +1,97 @@
+.. _user_eph:
+
+**********************
+Electron Phonon Matrix
+**********************
+
+*Modules*: :mod:`eph`, :mod:`pbc.eph`
+
+Introduction
+============
+Electron-Phonon (eph) Interaction describes the coupling between the electrons and the nuclear degree of freedom. First order eph matrix elements are most commonly used to compute electron/phonon transport properties. PySCF supports first order eph matrix calculation by either Coupled Perturbed Kohn Sham/Hartree Fock (CPKS/CPHF) or finite difference. In periodic system, only Gamma-point calculation with FFTDF is supported.
+
+A minimal example of analytical eph matrix calculation is as follows::
+
+    from pyscf import gto, scf, eph
+    mol = gto.M(
+    atom = [['O', [0.000000000000,  -0.000000000775,   0.923671924285]],
+            ['H', [-0.00000000000,  -1.432564848017,   2.125164039823]],
+            ['H', [0.000000000000,   1.432564848792,   2.125164035930]]],
+    basis = 'sto3g',
+    unit = 'Bohr'
+    )
+    # Note that this is a pre-computed relaxed molecule
+    mf = scf.RHF(mol).run()
+
+    myeph = eph.EPH(mf)
+    ephmat, omega = myeph.kernel()
+    print('eph matrix', ephmat)
+    print('phonon frequencies', omega)
+
+Alternatively, one can compute the matrix elements using finite difference::
+
+    from pyscf import gto, scf, eph
+    mol = gto.M(
+    atom = [['O', [0.000000000000,  -0.000000000775,   0.923671924285]],
+            ['H', [-0.000000000000,  -1.432564848017,   2.125164039823]],
+            ['H', [0.000000000000,   1.432564848792,   2.125164035930]]],
+    basis = 'sto3g',
+    unit = 'Bohr'
+    )
+    mf = scf.RHF(mol).run()
+
+    ephmat, omega = eph.eph_df.kernel(mf, disp=1e-4)
+    print('eph matrix', ephmat)
+    print('phonon frequencies', omega)
+
+Spin symmetry
+=============
+The eph module in PySCF supports mean field methods including Hartree Fock and DFT using either spin-restricted or spin-unrestricted reference.
+
+Analytical eph matrix elements
+------------------------------
+
+The module-level ``eph.EPH(mf)`` constructor can infer the correct method based
+on the level of symmetry-breaking in the mean-field argument.  For more explicit
```

Comment:
Symmetry breaking usually means a different thing, right? E.g. you can UHF that does not break spin symmetry i.e. is equivalent to RHF. Same for GHF.

Maybe replace "level of symmetry-breaking in the mean-field argument" with "type of the mf argument"?

Reply:
yes, replace with "type of mean-field argument (e.g. restricted, unrestricted, etc.)"

Reply:
This exact language occurs in a number of other places, originally introduced by me in the CC docs, so it'll have to be changed elsewhere.